styles/desktop-styles.css

```diff
@@ -0,0 +1,284 @@
+    margin: 0 auto;
+  }
+
+  .project-grid {
+    display: grid;
+    align-self: center;
+    justify-self: center;
+    grid-template-columns: repeat(auto-fit, minmax(14.8rem, 1fr));
+    grid-template-rows: repeat(auto-fit, minmax(auto, 1fr));
+    border-radius: 8px;
+    padding: 10px;
+    row-gap: 10px;
+    column-gap: 8px;
+  }
+
+  .project-container {
+    border-radius: 8px;
+    background-color: #ebf0ee;
+    border: 1px solid #c1c7d0;
+    aspect-ratio: 3/2;
+  }
+
+  .img-container {
+    background-color: #ebf0ee;
+    height: 12rem;
+    border-radius: 8px;
+  }
+
+  .project-details {
+    display: flex;
+    flex-direction: column;
+    color: #3a4a42;
+    background-color: #fff;
+    height: 13rem;
+    min-width: 0;
+    min-height: 0;
+    border-radius: 8px;
+  }
+
+  .project-details ul {
+    justify-content: center;
+    margin: 0;
+  }
+
+  .tag {
+    margin-top: 10px;
+    background-color: #ebf0ee;
+    padding: 7.7px;
+    height: 14px;
+    font-size: 10px;
+    white-space: nowrap;
+    margin-right: 4px;
+    border: none;
+    color: #3a4a42;
+    text-align: center;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    border-radius: 4px;
+  }
+
+  .btn-container {
+    display: flex;
+    justify-content: center;
+    margin-top: 15px;
+    margin-bottom: 20px;
+  }
+
+  /* .works .project-grid {
+    margin-left: 1em;
+  } */
+
+  .qualifications {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    text-align: center;
+    width: 100%;
+    height: 100%;
+    gap: 10px;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 1px solid #c1c7d0;
+    border-radius: 8px;
+    background-color: #ebf0ee;
+    padding: 2px;
+    padding-top: 20px;
+    margin-top: 20px;
+    width: 100%;
+    height: 316px;
+    max-width: 369px;
+    max-height: 316px;
+  }
+
+  .skill-tag-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+
+  .skill-tag {
+    background-color: #fff;
+    border: none;
+    color: #36b37e;
+    padding: 8px 10px;
+    text-align: center;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 12px;
+    margin-top: 8px;
+    margin-left: 2px;
+    margin-right: 2px;
+    border-radius: 8px;
+    list-style-type: none;
+  }
+
+  .skill-title {
+    margin: 0;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-image: url(../assets/about-me-background-desktop.jpg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 100%;
+    margin-top: 10em;
+  }
+
+  .about-me-container {
+    max-width: 768px;
+    height: auto;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 32px;
+    text-align: center;
+    padding-bottom: 2rem;
+
+    /* margin: 0 12.5rem; */
+  }
+
+  .contact {
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement as I mentioned above regarding your design not matching properly with the Figma design. Your design should start responsive from the `768px` screen size. Right now because as I can see you are not using the Correct `layout`, `media query`, and `spacing` around the About section and the font size of the About me section content is not correct. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table. 

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-17 04-43-08](https://github.com/BHObserver/portfolio/assets/66476329/e5c0f09c-ac6d-4fd7-908d-5bb3ee1a207e)

</td>
<td>

![Screenshot from 2023-06-17 04-43-25](https://github.com/BHObserver/portfolio/assets/66476329/e64bf279-9ece-4287-84bd-c1fd13ae6fc8)

</td>
</tr>
</table>


**Hint**: *Please use **%age** for adjusting width with `margin: auto;` for making your container responsive and aligned centre always according to the viewport of the screen. Container which holds the child's content like headings, sub-headings, text or form. Don't use the fixed width or absolute units*.